content/documentation/overview/index.adoc

```diff
@@ -197,7 +197,7 @@ This section lists all the validations that Kiali performs on all Istio configur
 |link:/files/validation_examples/109.yaml[109.yaml, window="_blank"]
 
 |(Deprecated) All routes should have weight
```

Comment:
I'm probably missing something here, but why is something under "deprecated" section fixed?

Reply:
Deprecated validations are those validations that Galley is going to take care to warn users. But since not many people have Galley up and running, we decided to keep them listed.
Perhaps, we should explain that right at the bottom of the section, shall we?

Reply:
why not many users have Galley up and running? is it disabled by default?

Reply:
It is not disabled by default but in different installations they have problems to having properly running. Using both `oc cluster up` and `minishift start` have problems. Although, maistra team sorted it out on their installer.

Reply:
is this a known/open issue in Istio repo? (shouldn't block this PR of course)

Reply:
It has been many, I guess they are close though. Sooner or later we will have to revisit this decision about assuming galley is not working in some environments.
I did submit a PR to Istio to fix this: https://github.com/istio/istio/pull/11586